src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+    {person.age && (
+      <p className="Person__age">
+        {`I am ${person.age}`}
+      </p>
+    )}
+    {person.isMarried ? (
+      <p className="Person__partner">
```

Comment:
Remember about DRY. This tag should be the same for both variants, so you need to change only content depending on `isMarried` status